src/NextjsRevalidation.ts

```diff
@@ -78,7 +92,7 @@ export class NextjsRevalidation extends Construct {
 
   private createFunction(): NodejsFunction {
     const nodejsFnProps = getCommonNodejsFunctionProps(this);
-    const fn = new NodejsFunction(this, 'Fn', {
+    const fn = new NodejsFunction(this, 'QueueRevalidationFn', {
```

Comment:
I made the id of this only `Fn` so that is deduplicates the term `Revalidation` in the name of the resource since the parent construct's name is `Revalidation`, it gets added twice. So I would prefer this to be `QueueFn` so that the ultimate resource name would be `RevalidationQueueFn` instead of `RevalidationQueueRevalidationFn`.

Reply:
With `Fn`, this is an example of a current resource name: `bui1-stickb-ui-NextjsRevalidationFn329BA096-CKuulcO3k6bB`

Reply:
Makes sense for sure. I'll change to `QueueFn` and `DynamoDBProviderFn`.

RE: names in general, for other projects I work on (unrelated to Next.js or this repo), I have function names like

```
prod-public-gateway-authorizer
prod-s3-publisher-handler 
prod-introspection-service
```

I normally prefix everything with a "stage" (even though I run different stages in different accounts, I still really like to be able to quickly / visually see "prod" or "dev", etc), then the names are very flat. I frankly don't know if this is "good" though, as I mentioned I'm not sure on best practice here, it just _looks_ tidy :).

For now I'm going to just strip "Revalidation" out, end up with (in my case) things like `Cdk2Stack-WebRevalidationQueueFn37B09F4F-zUasGdL53ANQ`